src/main/java/com/krafit/financialcalcs/LoanCalculatorApplication/LoanCalculatorRestController.java

```diff
@@ -0,0 +1,29 @@
+package com.krafit.financialcalcs.LoanCalculatorApplication;
```

Comment:
ExpectedPersonalLoanEMICalculationRestController

Reply:
changed to ExpectedPersonalLoanEMICalculationRestController